master/reference/advanced/etcd-rbac/overview.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Setting up etcd certificates for RBAC
+---
+
+When using etcd it is a good idea to protect the data stored there.  This is
+even more true when you have multiple components using a common etcd cluster.
+This guide and the linked guides are intended to help:
+- Restrict the operations that support staff are able to perform to minimize
+  accidental corruption or deletion of etcd data.
+- Lock down access to each tenants data, if the cluster is being used by
+  multiple orchestrators (Kubernetes, docker, Calico) or multiple tenants.
+- Restrict the read/write access of the varius Calico components as an
+  additional safety measure.
+
+## Overview
```

Comment:
How does this "Overview" section in this doc compare to the paragraph above?  Should we just drop the "Overview" sub-headings and put the intro blurb directly below the main doc heading?

especially true of this doc as this is an overview section of the overview doc.